README.md

```diff
@@ -49,11 +49,24 @@ chmod +x setup.sh
 ```
 Once the setup is completed (3-7 mins depending on how sprightly your local machine is feeling), you can view the argo-workflows UI at https://localhost:8443/argo/workflows/argo (the S in https is important and you'll need to accept the self-signed certificate). Make sure you are looking at the 'argo' namespace.
 
+### Run the workflow using yaml
+
 Then you can deploy the workflow and you should see it appear in the UI.
 ```
 kubectl -n argo create -f workflow.yml
 ```
 
+### Run the workflow using [Hera](https://hera.readthedocs.io/en/stable/)
+
+Ensure you have python and the hera framework installed.
+
+You can deploy all the templates and run the workflow using
+```
+python hera/nfs/workflow.py
```

Comment:
In pipekit hera things that I have (badly) written, the hera python doesn't end until the workflow has finished. It also splurges a link to the running workflow. Is that something that can be done here too, or is that all the pipekit sdk doing the heavy lifting? Example here: https://github.com/pipekit/examples/blob/main/examples/get-versions/workflow.py#L91

Reply:
That's the pipekit SDK doing heavy lifting. Unless @flaviuvadan tells me otherwise I don't think hera includes anything to help with that.

Reply:
get_workflow_link() doesn't work properly in our setup. I'll raise a hera bug.

Reply:
https://github.com/argoproj-labs/hera/issues/872